cranelift/codegen/Cargo.toml

```diff
@@ -55,6 +57,10 @@ std = []
 # compatibility as a no-op.
 core = []
 
+# Enable the `to_capstone` method on TargetIsa, for constructing a Capstone
+# context.
+disas = ["anyhow", "capstone"]
```

Comment:
Can `capstone::Error` wrap the error potentially raised by `write!`? That's currently the only thing holding back removing `anyhow` here.

Reply:
There's a `capstone::Error::CustomError(&'static str)` variant. Since there's no data attached to a `fmt::Error` (the `Display` impl currently always returns "an error occurred when formatting an argument") that would work. But yeah, I forgot about the `write` invocations.